app/models/user/User.scala

```diff
@@ -144,43 +144,44 @@ class UserDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext)
       )
     }
 
-  override protected def readAccessQ(requestingUserId: ObjectId) =
+  override protected def readAccessQ(requestingUserId: ObjectId): SqlToken =
     readAccessQWithPrefix(requestingUserId, SqlToken.raw(""))
 
   protected def readAccessQWithPrefix(requestingUserId: ObjectId, userPrefix: SqlToken) =
-    q"""(${userPrefix}_id in (select _user from webknossos.user_team_roles where _team in (select _team from webknossos.user_team_roles where _user = $requestingUserId and isTeamManager)))
-        or (${userPrefix}_organization in (select _organization from webknossos.users_ where _id = $requestingUserId and isAdmin))
-        or ${userPrefix}_id = $requestingUserId"""
+    q"""(${userPrefix}_id IN (SELECT _user FROM webknossos.user_team_roles WHERE _team IN (SELECT _team FROM webknossos.user_team_roles WHERE _user = $requestingUserId AND isTeamManager)))
+        OR (${userPrefix}_organization IN (SELECT _organization FROM webknossos.users_ WHERE _id = $requestingUserId AND isAdmin))
+        OR ${userPrefix}_id = $requestingUserId"""
   override protected def deleteAccessQ(requestingUserId: ObjectId) =
-    q"_organization in (select _organization from webknossos.users_ where _id = $requestingUserId and isAdmin)"
+    q"_organization IN (SELECT _organization FROM webknossos.users_ WHERE _id = $requestingUserId AND isAdmin)"
 
   private def listAccessQ(requestingUserId: ObjectId) = listAccessQWithPrefix(requestingUserId, SqlToken.raw(""))
   private def listAccessQWithPrefix(requestingUserId: ObjectId, prefix: SqlToken) =
     q"""(${readAccessQWithPrefix(requestingUserId, prefix)})
-        and
+        AND
         (
-          isUnlisted = false
-          or
-          ($requestingUserId in
+          NOT isUnlisted
+          OR
+          ($requestingUserId IN
             (
-              select u._id
-              from webknossos.users_ u join webknossos.multiUsers_ m on u._multiUser = m._id
-              where m.isSuperUser
+              SELECT u._id
+              FROM webknossos.users_ u
+              JOIN webknossos.multiUsers_ m ON u._multiUser = m._id
+              WHERE m.isSuperUser
             )
           )
         )"""
 
   override def findOne(id: ObjectId)(implicit ctx: DBAccessContext): Fox[User] =
     for {
       accessQuery <- readAccessQuery
-      r <- run(q"select $columns from $existingCollectionName where _id = $id and $accessQuery".as[UsersRow])
+      r <- run(q"SELECT $columns FROM $existingCollectionName WHERE _id = $id AND $accessQuery".as[UsersRow])
       parsed <- parseFirst(r, id)
     } yield parsed
 
   override def findAll(implicit ctx: DBAccessContext): Fox[List[User]] =
     for {
       accessQuery <- accessQueryFromAccessQ(listAccessQ)
-      r <- run(q"select $columns from $existingCollectionName where $accessQuery".as[UsersRow])
+      r <- run(q"SELECT $columns FROM $existingCollectionName WHERE $accessQuery".as[UsersRow])
```

Comment:
And in line 277 `utr.isteammanager` is spelled only with small letters. As you already changed the capitialization of other cooulmn names in this pr, would it be possible to make thi camel case here as well?

Reply:
And in lines 279+280 the `ON` keyword also needs to be capitalized:
```
            LEFT JOIN webknossos.user_team_roles utr ON utr._user = u._id
            LEFT JOIN webknossos.teams t ON t._id = utr._team -- should not cause fanout since there is only one team per team_role
```

The same goes for ll. 310-313